libs/application/templates/general-fishing-license/src/fields/Declined/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import { Box, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FieldBaseProps } from '@island.is/application/core'
+import { declined } from '../../lib/messages'
+import { WarningSection } from './WarningSection'
+
+export const Declined = ({ application }: FieldBaseProps): JSX.Element => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <>
+      <Box paddingTop={2}>
+        <Text>{formatMessage(declined.labels.helpText)}</Text>
+      </Box>
+      <Box marginTop={5} marginBottom={8}>
+        <WarningSection
+          application={application}
+          step={{
+            title: formatMessage(declined.labels.otherCountryTitle),
+            key: 'key',
```

Comment:
This key isnt used anywhere is it?

Reply:
And neither is the state, truthfully just copy pasted this from another application with payment hehe